curriculum/challenges/chinese/01-responsive-web-design/basic-html-and-html5/add-a-submit-button-to-a-form.chinese.md

```diff
@@ -2,29 +2,36 @@
 id: bad87fee1348bd9aedd08830
 title: Add a Submit Button to a Form
 challengeType: 0
-videoUrl: ''
-localeTitle: 向表单添加提交按钮
+videoUrl: 'https://scrimba.com/p/pVMPUv/cp2Nkhz'
+forumTopicId: 16627
+localeTitle: 给表单添加提交按钮
 ---
 
 ## Description
-<section id="description">我们在表单中添加一个<code>submit</code>按钮。单击此按钮会将表单中的数据发送到您使用表单的<code>action</code>属性指定的URL。这是一个示例提交按钮： <code>&lt;button type=&quot;submit&quot;&gt;this button submits the form&lt;/button&gt;</code> </section>
+<section id='description'>
+让我们来给表单添加一个<code>submit</code>提交按钮，当点击提交按钮时，表单中的数据将会被发送到<code>action</code>属性指定的地址上。
+例如：
+<code>&#60;button type="submit"&#62;this button submits the form&#60;/button&#62;</code>
```

Comment:
@raisedadead Should we translate the example snippet into Chinese as well (More specifically, "this button submits the form" here)? Personally, I'd consider that makes more sense by having it translated.